backend/danswer/datastores/qdrant/indexing.py

```diff
@@ -74,11 +75,22 @@ def index_chunks(
             for x in range(0, len(point_structs), DEFAULT_BATCH_SIZE)
         ]
         for point_struct_batch in point_struct_batches:
-            index_results = client.upsert(
-                collection_name=collection, points=point_struct_batch
-            )
+
```

Comment:
when does this fail? generally should probably do a backoff or sleep at least but not big deal